docs/self-hosted-appcircle/configure-server/minio-migration.md

```diff
@@ -0,0 +1,392 @@
+
+:::
+
+## Troubleshooting & FAQ
+
+### There is no space left on disk while migrating
+
+If you can connect to the server via SSH, you can delete the **newly created** single-node single drive MinIO volume to free up disk space for stable system operations.
+
+- List and filter the **`snsd`** volume from container volumes.
+
+<Tabs>
+  <TabItem value="docker" label="Docker" default>
+
+```bash
+docker volume ls | grep -i "snsd"
+```
+
+  </TabItem>
+
+  <TabItem value="podman" label="Podman">
+
+```bash
+podman volume ls | grep -i "snsd"
+```
+
+  </TabItem>
+</Tabs>
+
+- Remove the your project's **`snsd`** volume. For example;
+
+<Tabs>
+  <TabItem value="docker" label="Docker" default>
+
+```bash
+docker volume rm spacetech_minio_snsd_data
+```
+
+  </TabItem>
+
+  <TabItem value="podman" label="Podman">
+
+```bash
+podman volume rm spacetech_minio_snsd_data
+```
+
+  </TabItem>
+</Tabs>
+
+After that, you can cleanup the disk or add some more disk space for a successful migration.
+
+Check the **[prerequisites](#prerequisites)** section for the required disk space.
+
+If you want to go on without any migration and stay with an older configuration, you should follow the **[Staying with MNSD MinIO](#update-server)** section for configuration details.
+
+### Possible checks that can be done after migration
+
+In order to check if the migration is successful and the data is consistent, you can check some modules on the Appcircle dashboard.
+
+Below is a short list of common modules that can be checked.
+
+- [ ] Open a "Build Profile" and check the build logs.
+- [ ] Open a "Publish Profile" and check the publish logs.
+- [ ] Open a "Testing Distribution Profile" and check the app icons.
+- [ ] Open "Enterprise App Store" module and check the app icons.
+
+The migration command also prints output about the result of the migration operation on the command line.
+
+### When you get an error while or after migrating
+
+The migration operation does not delete the old MinIO volumes automatically. Your data before the migration is untouched and safe.
+
+So if you face any error while migrating or after migrating to the SNSD MinIO, you can revert to the old MNSD MinIO configuration.
+
+In this case follow the steps below to stay with MNSD MinIO configuration.
+
+- Log in to Appcircle server with SSH or remote connection.
+
+- Go to the `appcircle-server` directory.
+
+```bash
+cd appcircle-server
+```
+
+<SpacetechExampleInfo />
+
+- Stop the Appcircle server.
+
+```bash
+./ac-self-hosted.sh -n "spacetech" down
+```
+
+- Edit the `global.yaml` file of your project.
+
+```bash
+vi ./projects/spacetech/global.yaml
+```
+
+- Add the below configuration section to `global.yaml`, or change the value as below if the `minio` section existed before.
+
+```yaml
+minio:
+  type: mnsd
+```
+
+- Apply configuration changes.
+
+```bash
+./ac-self-hosted.sh -n "spacetech" export
+```
+
+- Start the Appcircle server.
```

Comment:
Precondition:

- **offline install/upgrade** user

Case: (_following documentation step by step_)

- Download the latest package.
- `down` -> `-i` -> `export`.
- run `minio-migrate` | failure.

Now the user comes this section since he got error while migrating.

He has the latest package but has no latest images. ❗

For a proxy registry or GCP artifact registry user there is no problem. `up` will pull the latest images. But for an offline install/upgrade user there will be problem.

💡 IMO it will be good to move the section below

```md
- Upgrade the container images.

./ac-self-hosted.sh -n "spacetech" upgrade
```

before "Migrating to SNSD MinIO (recommended)" "Staying with MNSD MinIO" tabs under Update Server title.

Reply:
Moved the section and added a caution box.